### PR TITLE
update readme to show how to use modular components in React and Angular

### PR DESCRIPTION
**Pull request by sarnikbixal**
<!-- Please feel free to remove whatever sections/lines in this aren’t relevant.

update readme to show how to use modular components in React and Angular

## Description

update readme to show how to use modular components in React and Angular

## Additional information

Include any of the following (as necessary): 

How do we set this on USWDS site? I am not sure if this Readme will update the same information on the uswds-site domain.

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [ ] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

#3575

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,6 +275,57 @@ Each component has a standardized interface that can be used to extend it furthe
 
 Some components have additional methods for manipulating specific aspects of them based on what they are and what they do. These can be found in the component's JS file.
 
+**If you are using am odern framework such as React/Angular you can import components and initialize it in your libraries dom ready lifecycle event.**
```
Comment (mejiaj) [inline README.md:278]: Typo `am odern framework`

File: README.md [hunk 2]
```diff
@@ -275,6 +275,57 @@ Each component has a standardized interface that can be used to extend it furthe
...
+```
+
+React hooks example:
+
+```js
+function App() {
+  const ref = document.body;
+
+  useEffect(() => {
+     // inititalize
```
Comment (mejiaj) [inline README.md:294]: Typo on `initialize`

File: README.md [hunk 3]
```diff
@@ -275,6 +275,57 @@ Each component has a standardized interface that can be used to extend it furthe
...
+     characterCount.on(ref); // default ref is document.body, if you want to use default you do not bave to pass arguments
```
Comment (mejiaj) [inline README.md:295]: Typo `default you do not bave to pass arguments`

File: README.md [hunk 4]
```diff
@@ -275,6 +275,57 @@ Each component has a standardized interface that can be used to extend it furthe
...
+  })
+}
+```
+
+Angular example:
+
+```js
+export class App implements OnInit {
+  constructor() {
+    this.ref = document.body; // default ref is document.body, if you want to use default you do not bave to pass arguments
```
Comment (mejiaj) [inline README.md:312]: Typo `you do not bave to pass arguments`

File: README.md [hunk 5]
```diff
@@ -275,6 +275,57 @@ Each component has a standardized interface that can be used to extend it furthe
...
+  }
+
+  ngOnInit() {
+    // inititalize
```
Comment (mejiaj) [inline README.md:316]: Typo ` // inititalize`

**Review by mejiaj — CHANGES_REQUESTED** [review]
Some typos, otherwise LGTM.

**Review by sarnikbixal — COMMENTED** [review]
fixed typos

**Review by mejiaj — APPROVED** [review]

**Pull request merged**